fastlane/Fastfile

```diff
@@ -40,11 +40,15 @@ platform :ios do
     )
   end
 
+  lane :sentry do |options|
+    sh "cd .. && export APPCENTER_ACCESS_TOKEN=#{ENV['IOS_APPCENTER_API_TOKEN']} && export BUILD=`jq -r .build package.json` && export VERSION=`jq -r .version package.json` && export CODE_PUSH_LABEL=$(curl -sS \"https://codepush.appcenter.ms/v0.1/public/codepush/update_check?app_version=${VERSION}&deployment_key=${CODEPUSH_KEY}\" | jq -r '.update_info.label') && export RELEASE_NAME=${VERSION}-ios+codepush:${CODE_PUSH_LABEL} && export DIST=${BUILD}-ios && npx sentry-cli react-native appcenter #{ENV['APPCENTER_ORGANISATION']}/#{ENV['APPCENTER_IOS_APP_ID']} ios ./build/CodePush --deployment #{ENV['CODEPUSH_DEPLOYMENT_NAME']} --dist ${DIST} --release-name ${RELEASE_NAME}"
```

Comment:
Eventually, remove curl command and use `appcenter-cli`, ex:

```bash
$ npx appcenter codepush deployment history -a pass-Culture/passculture-staging-android CodePush
```

Try to check if filtering works with `-v | --version`